src/client/containers/layout.js

```diff
@@ -100,26 +114,28 @@ class Layout extends React.Component {
 						<NavDropdown
 							eventKey={1}
 							id="create-dropdown"
+							open={this.state.menuOpen}
```

Comment:
A good practice, when you add an item to the state, is to always give it an initial value.
That way, someone else reading your code will know what the initial state is and what to expect.

In this case, you'll want to add the initial value for `menuOpen` in the constructor, where `this.state` is defined